modin/pandas/series.py

```diff
@@ -81,6 +81,7 @@ class Series(BasePandasDataset):
 
     _pandas_class = pandas.Series
     __array_priority__ = pandas.Series.__array_priority__
+    _accessors = {"dt", "cat", "str", "sparse"}
```

Comment:
@jbrockmendel in order to reuse `_register_accessor` function from pandas, it's needed to add `_accessors` in Modin objects. 

Can you tell me what this attribute is for and maybe you can also tell me how to properly integrate it into Modin?

Reply:
its used to control what shows up in `__dir__` via _dir_deletions and _dir_additions